signals/example_model.py

```diff
@@ -151,8 +151,12 @@ def main():
     # choose data as of most recent friday
     last_friday = datetime.now() + relativedelta(weekday=FR(-1))
     date_string = last_friday.strftime('%Y-%m-%d')
-
-    live_data = full_data.loc[date_string].copy()
+    
+    try:
+        live_data = full_data.loc[date_string].copy()
+    except KeyError as e:
+        print(f"No ticker on {e}")
+        live_data = pd.DataFrame(columns=full_data.reset_index().columns).set_index('date')
```

Comment:
This could be simpler.
```suggestion
        live_data = full_data.iloc[:0].copy()
```

Reply:
Thanks @jeethu . Committed.